spec/lib/govuk_publishing_components/presenters/single_page_notification_button_helper_spec.rb

```diff
@@ -52,9 +52,9 @@
       expect(single_helper.skip_account_param).to eq("single_page_subscription")
     end
 
-    it "sets skip_account when skip_account is a string value of true" do
+    it "does not set skip_account when skip_account is a string value of true" do
```

Comment:
I feel like you can probably take this test out? I'm not sure it adds anything to the coverage or the understanding of the component's behaviour.

Reply:
Fair enough, have removed 👍